src/frontend/src/handler/create_mv.rs

```diff
@@ -145,6 +146,16 @@ pub async fn handle_create_mv(
         (table, graph)
     };
 
+    let _job_guard = StreamingJobGuard::new(
```

Comment:
Can we make it a method of `Tracker`? 🥵 We can pass `self` to create a guard there.

Reply:
okk, I will change it.